accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloAttributeStore.scala

```diff
@@ -111,18 +114,22 @@ class AccumuloAttributeStore(val connector: Connector, val attributeTable: Strin
 
   def layerIds: Seq[LayerId] = {
     val scanner = connector.createScanner(attributeTable, new Authorizations())
-    scanner.iterator
+    val retval = scanner.iterator
       .map { kv =>
         val List(name, zoomStr) = kv.getKey.getRow.toString.split(SEP).toList
         LayerId(name, zoomStr.toInt)
       }
       .toList
       .distinct
+
+    scanner.close ; retval
```

Comment:
quite important changes; @lossyrob I can take them into account too for hbase backend